viper.go

```diff
@@ -1348,8 +1348,8 @@ func (v *Viper) find(lcaseKey string, flagDefault bool) interface{} {
 				s = strings.TrimSuffix(s, "]")
 				res, _ := readAsCSV(s)
 				return cast.ToIntSlice(res)
-			case "stringToString":
-				return stringToStringConv(flag.ValueString())
+			case "stringToString", "stringToInt":
+				return stringToInterfaceCong(flag.ValueString())
```

Comment:
`Conv` here as well

Reply:
sorry I mistakenly didn't pushed the changes